### DISCUSSION TITLE
What is the optimal battery limit for a MacBook?

### DISCUSSION THREAD

**Discussion by thirteenkai** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1373] [Q&A] [answered]
I use AlDente to limit charging to 50%. I mainly use the power supply at work, so I don't have to worry about running out of battery. Is 50% a good choice, or is 80% better?

**Comment by MatthiasKerbl** [comment]
Hi @thirteenkai,

Unfortunately, different research papers show slightly different results. Therefore, we do not recommend a single battery percentage as the best one for battery longevity. Apple recommends 80%, which is a good starting point. However, some research indicated even better results between 40 and 60%. With charge limits that low, you need to calibrate your battery regularly.

Best,
Matthias

**Comment by thirteenkai** [comment, marked as answer]
Thank you for your answer. I plan to limit it to 50% and calibrate once a week.
